src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/LinkDefendantTest.java

```diff
@@ -41,7 +42,24 @@ public void shouldBeAbleToSuccessfullyLinkDefendantV1() {
     }
 
     @Test
-    @Ignore("Disabled due to not using CCD integration currently")
+    public void shouldBeAbleToSuccessfullyLinkDefendantV2() {
+        Claim createdCase = commonOperations.submitClaim(
+            claimant.getAuthorisation(),
+            claimant.getUserDetails().getId()
+        );
+
+        User defendant = idamTestService.createCitizen();
+
+        Claim claim = linkDefendantV2(defendant)
```

Comment:
how do we know what claim should be linked to defendant? He might have more than 1 claim, can't he?

Do we have any test to cover this scenario?

Reply:
defendant should be linked to all outstanding unlinked claims, a test should be added for this